python/tvm/auto_scheduler/auto_schedule.py

```diff
@@ -156,16 +159,41 @@ def __init__(
         )
 
 
+def create_task(func, args, target, target_host=None, hardware_params=None):
```

Comment:
I'm thinking about what's the different of calling `SearchTask()` directly, seems they just have different input parameters.

I notice that AutoTVM used an api of `autotvm.task.create()`, should we keep them the same?

Reply:
Python does not support constructor overloading, so I have to create a new function for this.

I think both `auto_scheduler.create_task` and `auto_scheduler.task.create` are okay.